### PR TITLE
Fix the property `lithology type` and `in-canopy sky`

### PR DESCRIPTION
**Pull request by junrongYu**
The `lithology type` in `plot soil description` should be divided into `rock outcrop lithology` and `surface strew lithology`.

Fixes: #123

### CODE REVIEW

**Comment by junrongYu** [comment]
Currently in `rock outcrop lithology` and `surface strew lithology`, there are multiple source, definition, and feature type integrated from `lithology type`, need to be confirmed.

**Comment by junrongYu** [comment]
The definition and feature type for `in-canopy sky` need to be confirmed.

**Comment by junrongYu** [comment]
Keep both feature types for both `lithology`.

**Comment by junrongYu** [comment]
The feature type for `ics` is `plant individual`.

File: vocab_files/observable_property_concepts/in-canopy-sky.ttl [hunk 1]
```diff
@@ -6,13 +7,17 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 <https://linked.data.gov.au/def/test/dawe-cv/85bdd25a-fa08-49de-9f0b-98895cb79aa6>
     a skos:Concept ;
+    dcterms:source "Laws M, McCallum K, Bignall J, O’Neill S, Sparrow B. (unpublished draft) ‘Fire module’ in ….. (eds) Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide." ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:altLabel "in canopy sky (ics) status" ;
```
Comment (edmondchuc) [inline vocab_files/observable_property_concepts/in-canopy-sky.ttl:12]: I would have the pref label as `in-canopy sky` or `in-canopy sky status` and the alt label as `ICS`

**Review by edmondchuc — APPROVED** [review]

**Pull request merged**